tests/test_buku.py

```diff
@@ -562,6 +562,28 @@ def test_import_md(tmpdir, newtag, exp_res):
     res = list(import_md(p.strpath, newtag))
     assert res[0] == exp_res
 
+@pytest.mark.parametrize(
+    "newtag, exp_res",
+    [
+        (None, ("http://example.com", "text1", ",", None, 0, True, False)),
+        ("tag1", ("http://example.com", "text1", ",tag1,", None, 0, True, False)),
+    ],
+)
+def test_import_rss(tmpdir, newtag, exp_res):
+    from buku import import_rss
+
+    p = tmpdir.mkdir("importrss").join("test.rss")
+    p.write(
+        """<feed xmlns="http://www.w3.org/2005/Atom">
+        <title>Bookmarks</title>
+        <entry>
+            <title>text1</title>
+            <link href="http://example.com"/>
+        </entry>
+    </feed>"""
+    )
```

Comment:
The indentation looks off here. Either use [`dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent), or separate literals for each line.

Reply:
I'm not sure what you mean. Should there be no indentation?

Reply:
You're trying to have a structurally indented XML, but having to accommodate for the quotation and the external code breaks away from that.

You can adjust your multiline text for proper indentation like this:
```py
    p.write(dedent("""\
        <feed xmlns="http://www.w3.org/2005/Atom">
            <title>Bookmarks</title>
            <entry>
                <title>text1</title>
                <link href="http://example.com"/>
            </entry>
        </feed>
    """))
```
This allows to incorporate the heredoc indentation into the file without breaking the inner indents of the text, while stripping the extra indentation levels from the actual output.

Reply:
Done